### ISSUE TITLE
A11y_code.visualstudio.com(CELA-P1)_Get Started_Keyboard: Focus is not properly defined on "Python" Extension

### ISSUE THREAD

**Issue by Arjun553** [microsoft/vscode-docs#2646]
**Check out Keros!** - Identify accessibility bugs before check-in and make bug fixing faster and easier.
**Environment Details:**
code.visualstudio.com/docs
Browser: Edge

**Repro Steps:**
1)Hit the URL:https://code.visualstudio.com/docs
2)Verify that bold frame(dotted or plain) is there to distinguish between the focused element and unfocused elements on the page.

**Actual Result:**
Focus is not properly defined on "Python" Extension

**Expected Result:**
Focus should be  properly defined on "Python" Extension so that the user who cant able to see clearly can easily understand the issue

**User Impact:**
Keyboard users will face difficulty in understanding and accessing the controls if   Focus is not properly visible on "Python" Extension

**Mas reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}
[Bug_2646_Keyboard.pptx](https://github.com/microsoft/vscode-docs/files/3161779/Bug_2646_Keyboard.pptx)

**Comment by gregvanl** [comment]
I think the issue is that the extension Marketplace tiles focus is not very visible.

![image](https://user-images.githubusercontent.com/12818376/58136248-48758f00-7be2-11e9-8c71-3c88a2c3d404.png)

**Comment by gregvanl** [comment]
@Arjun553 Fixed

![image](https://user-images.githubusercontent.com/12818376/59232192-462aa300-8b98-11e9-8eaf-db696566d5a1.png)

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
